ocr/app.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pathlib
+import ocrmypdf
+import sys
+
+
+deploy_dir = os.environ.get("DEPLOY_DIR") or pathlib.Path(__file__).parent
+sys.stdout = open('/dev/null', 'w')
```

Comment:
just curious why we're doing this.

Reply:
This is because writing to the output in the enclave would fail.